### PR TITLE
Add Content Via Seed

### PR DESCRIPTION
**Pull request by michaelwehrley**
* Set up seed file
* Add html and jpg and pngs for images
* Finished Lesson 26
* Finished Lesson 27
* Finished lesson 30

[#103526072]

### CODE REVIEW

File: app/models/bit_core/html_slide.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module BitCore
+  # A Slide with html content.
+  class HtmlSlide < Slide
+    serialize :options
+
+    def render_body
+      body.nil? ? "" : body
```
Comment (ericcf) [inline app/models/bit_core/html_slide.rb]: Are you planning to sanitize the HTML here or in a different part of the code?

Comment (michaelwehrley) [inline app/models/bit_core/html_slide.rb]: When `render_body` gets called in the view (in TFD Engine), but here might make more sense if it has access to the helpers.  I maybe can require `action_view` etc.  Thanks!

Comment (ericcf) [inline app/models/bit_core/html_slide.rb]: I suggest you put the logic here so you don't have to remember to sanitize everywhere it gets called. You don't need to require action view. Here's one way of doing it: https://github.com/NU-CBITS/munoz/blob/master/app/serializers/what_if_page_serializer.rb

Comment (michaelwehrley) [inline app/models/bit_core/html_slide.rb]: :jack_o_lantern:

**Pull request closed without merging**